ops/test/persistent/main.tf

```diff
@@ -28,8 +29,8 @@ module "bastion" {
   resource_group_location = data.azurerm_resource_group.test.location
   resource_group_name     = data.azurerm_resource_group.test.name
 
-  virtual_network_name = "${local.name}-${local.env}-network"
-  subnet_cidr          = ["10.3.253.0/27"]
+  virtual_network_name = module.vnet.network_name
+  subnet_cidr          = [cidrsubnet(local.network_cidr, 11, 2024)] # ["10.3.253.0/27"]
```

Comment:
I know this is the actual way the CIDR subnet is calculated, but it's not super human-readable - I had to verify the subnet in `terraform console`. We can agree that this process just is what it is (i.e. bit math) or maybe we could do something like `replace(local.network_cidr, "0.0/16", "253.0/27")`?

Reply:
Given that we're looking at re-IPing everything sometime soon, possibly into smaller network ranges... I dunno, I didn't want to leave hard-codes, but this is realistically just a different type of hard-code.

Reply:
Resolved: we moved this specific calculation into the `bastion_host` module, and put comments on all the cidrsubnet calls to make it clear what we were doing in each case.